core/utils/assets-loader.php

```diff
@@ -0,0 +1,84 @@
+<?php
+namespace Elementor\Core\Utils;
+
+use Elementor\Core\Base\Module;
+use Elementor\Plugin;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+/**
+ * Elementor assets loader.
+ *
+ * A class that is responsible for conditionally enqueuing styles and script assets that were pre-enabled.
+ *
+ * @since 3.1.0
+ */
+class Assets_Loader extends Module {
+	private $assets;
+
+	public function get_name() {
+		return 'assets-loader';
+	}
+
+	private function init_assets() {
+		$this->assets = [
+			'styles' => [
+				'e-animations' => [
```

Comment:
Tty to convert each asset to a class

Reply:
As agreed, will be done in a different scope.